internal/provider/namespace_resource.go

```diff
@@ -312,8 +313,13 @@ func (r *namespaceResource) Create(ctx context.Context, req resource.CreateReque
 		}
 		mtls := &namespacev1.MtlsAuthSpec{}
 		if plan.AcceptedClientCA.ValueString() != "" {
+			certs, err := base64.StdEncoding.DecodeString(plan.AcceptedClientCA.ValueString())
+			if err != nil {
+				resp.Diagnostics.AddError("Invalid (base64 encoded) accepted_client_ca", err.Error())
+				return
+			}
 			mtls.Enabled = true
-			mtls.AcceptedClientCa = plan.AcceptedClientCA.ValueString()
+			mtls.AcceptedClientCa = certs
```

Comment:
is this change a part of the move to enums? If not, given the large impact to enums, could we isolate this into a different PR?

Reply:
ack - TY